doc/plutip-testing.md

```diff
@@ -26,13 +27,43 @@ CTL depends on a number of binaries in the `$PATH` to execute Plutip tests:
 - [`ogmios`](https://ogmios.dev/)
 - [`kupo`](https://cardanosolutions.github.io/kupo/)
 
-All of these are provided by CTL's `overlays.runtime` (and are provided in CTL's own `devShell`). You **must** use the `runtime` overlay or otherwise make the services available in your package set (e.g. by defining them within your own `overlays` when instantiating `nixpkgs`) as `purescriptProject.runPlutipTest` expects all of them.
+All of these are provided by CTL's `overlays.runtime` (and are provided in CTL's own `devShell`). You **must** use the `runtime` overlay or otherwise make the services available in your package set (e.g. by defining them within your own `overlays` when instantiating `nixpkgs`) as `purescriptProject.runPlutipTest` expects all of them; an example of using CTL's overlays is in a [`ctl-scaffold` template](../templates/ctl-scaffold/flake.nix#L35).
 
-The services are NOT run by `docker-compose` as is the case with `launchCtlRuntime`: they are started and stopped on each CTL `ContractTest` execution by CTL itself.
+The services are NOT run by `docker-compose` (via `arion`) as is the case with `launchCtlRuntime`: instead they are started and stopped on each CTL `ContractTest` execution by CTL itself.
+
+If you have based your project on the [`ctl-scaffold` template](../templates/ctl-scaffold) then you have two options to run Plutip tests:
+1. `nix develop` followed by `npm run test` (recommended for development)
+2. `nix run .#checks.x86_64-linux.ctl-scaffold-plutip-test`
+   * where you'd usually replace `x86_64-linux` with the system you run tests on
+   * and `ctl-scaffold-plutip-test` with the name of the plutip test package for your project;
+   * note that compilation of your project via Nix will fail in case there are any warnings, so you'd need to either fix or temporarily disable them.
```

Comment:
Why this warning?

Reply:
Mainly because `npm run test` will run plutip tests even if there are warnings, so compared to to it `nix run` refusing to run tests might come as a surprise if you just want to quickly run some tests. Compilation with `nix run` also starts from scratch and is much slower, so this part of doc manages some expectations people might have.